File: khalti-android/src/main/java/khalti/widget/ButtonStyle.java
package khalti.widget;


import android.support.annotation.Keep;

@Keep
public enum ButtonStyle {
    BASIC(-1),
    EBANKING_DARK(0),
    EBANKING_LIGHT(1);

    private final int id;

    ButtonStyle(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }
}

File: khalti-android/src/main/java/com/khalti/widget/ButtonStyle.kt
package com.khalti.widget


import androidx.annotation.Keep

@Keep
enum class ButtonStyle(val value: Int) {
    BASIC(-1),
    KHALTI(0),
    EBANKING(1),
    MOBILE_BANKING(2),
    SCT(3),
    CONNECT_IPS(4),
}